backend/Cargo.toml

```diff
@@ -144,8 +144,7 @@ swc_ecma_ast = "0.106.2"
 base64 = "0.21.0"
 hmac = "0.12.1"
 sha2 = "0.10.6"
-sqlx = { version = "0.6", features = [
-    "offline",
+sqlx = { git = "https://github.com/mrl5/sqlx", rev = "16e4c9a8f380214ad61b1744d3e1f5fa866ffbcc", features = [
```

Comment:
notice that I'm temporarily using my custom patch https://github.com/mrl5/sqlx/commit/16e4c9a8f380214ad61b1744d3e1f5fa866ffbcc (that's basically copy paste from sqlx v0.6.3)

I hope it will be merged upstream and available in v0.7.1